ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -125,12 +125,12 @@ tests:
 - as: rosa-sts-ad-critical-high-f3
   cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-terraform-qe
```

Comment:
Have you tried if  `cluster_profile: aws-terraform-qe` can work well now?
I used `aws-qe` because I always met oidc-config failure when using aws-terraform-qe.

Reply:
it can't - on the other hand, `aws-qe` does not work for me either.
so I assume it is related to rehearse which is not working for slack notification.

Reply:
also regarding `oicd-config`: no issue seems to be detected from the CI checks, except of `tide` they are all passing.

Reply:
tide won't check the oidc-config issue. Oidc-config issue on happens when we run the job, and it happens on setup step